flutter-idea/src/io/flutter/run/OpenDevToolsAction.java

```diff
@@ -75,8 +76,13 @@ public void actionPerformed(@NotNull final AnActionEvent event) {
 
       final String serviceUrl = myConnector != null && myConnector.getBrowserUrl() != null ? myConnector.getBrowserUrl() : null;
 
+      FlutterSdk flutterSdk = FlutterSdk.getFlutterSdk(project);
+      if (flutterSdk == null) {
+        return;
+      }
```

Comment:
Do you only support launching DevTools for For Flutter projects? In VS Code we let you use it for Dart-only projects where there might not be a Flutter SDK (so we gate the new URLs on the Dart SDK version rather than Flutter).

Reply:
This is only for Flutter projects. The Dart plugin is managed by JetBrains and doesn't open the same panels